testing/e2e/test_cases/workload_create_test.go

```diff
@@ -0,0 +1,73 @@
+//go:build integration
+// +build integration
+
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/testing/e2e/helpers"
+	it "github.com/vmware-tanzu/apps-cli-plugin/testing/e2e/suite"
+)
+
+func TestCreateFromGitWithAnnotations(t *testing.T) {
+	testSuite := it.CommandLineIntegrationTestSuite{
+		{
+			Name:         "Create workload with valid name from a git repo and annotations",
+			WorkloadName: "test-create-git-annotations-workload",
+			Command: func() it.CommandLine {
+				c := *it.NewCommandLine(
+					"tanzu", "apps", "workload", "create", "test-create-git-annotations-workload",
+					"--annotation=min-instances=2", "--annotation=max-instances=4",
+					"--app=test-create-git-annotations-workload",
+					"--git-tag=tap-1.2",
+					"--git-repo=https://github.com/sample-accelerators/spring-petclinic",
+					namespaceFlag,
+					"--type=web",
+					"--yes",
+				)
+				return c
+			}(),
+			ExpectedRemoteObject: helpers.GetWorkloadFromFile(t, filepath.Join(expectedBasePath, "create-wld", "test-create-git-annotations-workload.yaml")),
```

Comment:
isnt it better to build workload object here instead of reading from file? To follow the test expectations I need to open another file. for readability sake i would stick to building the workload object here,

Reply:
It was to avoid long files but yes it is easier to have it in the same one, the option to load from file remains but changed to create the wld inline